pkg/cli/secretsencrypt/secrets_encrypt.go

```diff
@@ -12,51 +12,45 @@ import (
 	"github.com/erikdubbelboer/gspt"
 	"github.com/rancher/k3s/pkg/cli/cmds"
 	"github.com/rancher/k3s/pkg/clientaccess"
-	"github.com/rancher/k3s/pkg/daemons/config"
 	"github.com/rancher/k3s/pkg/secretsencrypt"
 	"github.com/rancher/k3s/pkg/server"
 	"github.com/rancher/k3s/pkg/version"
 	"github.com/urfave/cli"
 	"k8s.io/utils/pointer"
 )
 
-func commandPrep(app *cli.Context, cfg *cmds.Server) (config.Control, *clientaccess.Info, error) {
-	var controlConfig config.Control
+func commandPrep(app *cli.Context, cfg *cmds.Server) (*clientaccess.Info, error) {
 	var err error
```

Comment:
I don't think we need this forward declaration now that you added the colon below.